modules/acquia_cms_headless/tests/src/Functional/DashboardTestBase.php

```diff
@@ -4,14 +4,11 @@
 
 use Acquia\DrupalEnvironmentDetector\AcquiaDrupalEnvironmentDetector;
 use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
-use Drupal\Tests\acquia_cms_headless\Traits\DashboardSectionTrait;
 
 /**
  * Base class for the HeadlessDashboard web_driver tests.
  */
-abstract class DashboardWebDriverTestBase extends WebDriverTestBase {
-
-  use DashboardSectionTrait;
+abstract class DashboardTestBase extends WebDriverTestBase {
```

Comment:
instead of DashboardTestBase , lets give the name HeadlessTestBase so that it can be used everywhere instead of only on Dashboard.
 @apathak18 @rajeshreeputra